wrappers/python/openmm/app/internal/amber_file_parser.py

```diff
@@ -558,7 +558,7 @@ def getCMAPDihedrals(self):
         self._cmapList=[]
         forceConstConversionFactor = (units.kilocalorie_per_mole).conversion_factor_to(units.kilojoule_per_mole)
         for ii in range(0,len(cmapPointers),6):
-            if any([int(cmapPointers[ii+jj])<0 for jj in range(5)]):
+            if any(int(cmapPointers[ii+jj])<0 for jj in range(5)):
```

Comment:
% `python3.14 -m timeit "any([x for x in range(1000)])"`
```
10000 loops, best of 5: 24.1 usec per loop
```
% `python3.14 -m timeit "any(x for x in range(1000))"`
```
1000000 loops, best of 5: 256 nsec per loop
```
% `python3.14 -m timeit "all([x for x in range(1000)])"`
```
10000 loops, best of 5: 24.2 usec per loop
```
% `python3.14 -m timeit "all(x for x in range(1000))"`
```
1000000 loops, best of 5: 237 nsec per loop
```